privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -15,7 +15,7 @@ myApp.controller("tokenDetailController", function ($scope,
     $scope.selectedRealms = {};
     $scope.newUser = {user: "", realm: $scope.defaultRealm};
     $scope.loggedInUser = AuthFactory.getUser();
-    $scope.machinesPerPage = 5;
+    $scope.machinesPerPage = 15;
```

Comment:
Hi,

Currently I use only the ssh key administration.
But for 50-70 server.
There are then 5 entries per page very little.

Best regards
Marc